test/translate-test.js

```diff
@@ -29,21 +29,28 @@ describe('translate', () => {
 
     it('should include strides data', () => {
       translateResponse(patientData, stridesData);
-      expect(patientData.some(resource => resource.resourceType === 'DiagnosticReport' && resource.conclusionCodes?.some(cc => cc.coding.some(coding => coding.display?.includes('STRIDES Code'))))).to.be.true;
-      expect(patientData.some(resource => resource.resourceType === 'Procedure')).to.be.true;
+      expect(patientData.some(resource => resource.resourceType === 'DiagnosticReport' &&
+                              resource.code.coding.some(coding => coding.code === '65753-6' && coding.system === 'http://loinc.org') &&
+                              resource.conclusionCodes?.some(cc => cc.coding.some(coding => coding.system == 'urn:uuid:90915bcf-353c-49e1-b65e-0464798baa77')))
```

Comment:
Why are we asserting that the conclusion code system is `urn:uuid:90915bcf-353c-49e1-b65e-0464798baa77` as opposed to the actual SNOMED-CT system code? Is this just to prove that the data came from STRIDES?

Reply:
We can. The purpose here is to assume that the strides code is added. We don't quite care what the exact snomed code is. If we change the mocked patientData, then the mapped snomed code would be different but this uuid code remains the same.

Reply:
Got it, that makes sense to me.